docs/pipeline/automation/README.md

```diff
@@ -95,11 +95,13 @@ See [rnasum.md](rnasum.md)
 
 ### Somalier
 
+_Call Somalier through Holmes orchestration pipeline_
+
 ```
 aws lambda invoke --profile prodops \
```

Comment:
Exciting! Does the equivalent somalier relate function name exist? I think the relate payload will be more useful for users. Extract will almost always happen automatically, but there may be times a user wants to relate two bam files that would otherwise not be checked automatically.

Reply:
This one; is one for @andrewpatto

Reply:
Hold on. Alexis; @alexiswl do you mean Holmes "check"? It is there, scroll down a bit in readme. However, not sure, Holmes check call is `somalier relate ...` equivalent.

And. For Portal use case, mostly is per BAM check or extract call, sequentially. 
So I deduced that at Portal side.
If we wish multiple BAMs payload, I was thinking; we might as well use Holmes step function call directly; instead of proxying call through Portal Lambda.

Reply:
Yep see that is there, thank you